.travis.yml

```
@@ -13,13 +13,8 @@ install:
 - pip install -r test/requirements-dev.txt
 - find . -name "requirements.txt" -exec pip install -r "{}" \;
 
-before_script:
-# To ignore a directory, we can just delete the directory
-# so that treon doesn't see it.
-- xargs -t rm -rf < test/ignore
-
 script:
 # Test that notebooks run successfully
-- find "$PWD" -name "*.ipynb" -exec sh test/test_notebook.sh "{}" +
+- N=`awk '{printf " --exclude " $0}' test/ignore` treon ${!N}
```

Nit: I think `$( ... )` is slightly better (safer?) than backticks
```suggestion
- N=$(awk '{printf " --exclude " $0}' test/ignore) treon ${!N}
```

---

Also, just curious, I haven't seen this `${!N}` notation before, what does that do?

---

Agreed. I pushed this to see if it would pass on Travis (I was having problems locally...) and ended up replacing it with this:

    exclude="$(awk '{printf " --exclude " $0}' test/ignore)" eval "treon $exclude"

The `!` was an attempt to use indirect expansion in bash, which didn't work well.